docs/source/general/callback.rst

```diff
@@ -0,0 +1,46 @@
+########
+Callback
+########
+
+.. _callback:
+
+**************
+Flash Callback
+**************
+
+:class:`~flash.data.callback.FlashCallback` is an extension of the PyTorch Lightning `Callback <https://pytorch-lightning.readthedocs.io/en/latest/extensions/callbacks.html>`__.
+
+A callback is a self-contained program that can be reused across projects.
+
+Flash and Lightning have a callback system to execute callbacks when needed.
+
+Callbacks should capture any NON-ESSENTIAL logic that is NOT required for your lightning module to run.
+
```

Comment:
How are callbacks passed in flash? only using trainer? let's add that

Reply:
Done !